keyboards/index.php

```diff
@@ -9,9 +9,35 @@
   use Keyman\Site\com\keyman\templates\Foot;
   use Keyman\Site\com\keyman\Locale;
 
+  // Of array of strings at top of file
+  // by msgid 
+  $keyboardIndexStrings = Locale::localize('keyboards', [
```

Comment:
This all feels very fragile -- arrays across multiple files which have to match up. A lot of duplication of these strings -- 6 times:
* twice in keyboards-en.po
* once each in keyboards-es-ES.po, keyboards-fr-FR.po
* twice in keyboards/index.php -- here and where referenced.

This will need automatic validation to keep in sync or it's going to get very hard to maintain. (Why do we need the array defined here as well as in the keyboards-en.po file? Can't we just use the list in that file as our starting point?)

It may be better to use an identifier which is not the whole string? e.g. I see we have also 'keyboard search' 'keyman keyboard search' 'keyboard search%s', 'search', 'new search' ... lots of repetition here and risk of confusion!

Reply:
It might be sensible to establish a convention for a shorthand reference e.g. `$_` rather than a long name like `$keyboardIndexStrings`

Reply:
Based on :duck: -
* Replaced $keyboardIndexStrings array with `_()` alias
* Updated `Locale::localize()` so it doesn't restore textdomain